prefect_twitter/media.py

```diff
@@ -69,3 +69,46 @@ def example_media_upload_flow():
     media = await to_thread.run_sync(partial_media)
     media_id = media.media_id
     return media_id
+
+
+@task
+async def get_media_upload_status(
+    media_id: int, twitter_credentials: "TwitterCredentials"
+) -> int:
```

Comment:
Based on the docstrings and the code, it looks like this annotation either needs to be updated or removed so that the return type hinting is accurate.